changeInterface gets mode=remove back

richtong opened the pull request:
Updated the error messages as well, but the dns additions removed the "remove" option. I put back in. Also there is a fix for static bug as well you should pull that in.

richtong committed:
changeInterface gets mode=remove back

JoeKuan commented:
Oops. I have added the mode=remove. Also fixed the remove mode on static interface

richtong commented:
Thanks! Also there is a fork of your stuff that appears to fix a bug. Did u see that. Nice work by the way!

Thanks,
Rich

Nothing in the world can take the place of persistence... The slogan "press on" has solved and will always solve the problems of the human race. Locomotive Engineers Monthly. November 1914.

http://tongfamily.com | @richtong | http://fb.me/richtong1 | +1-206-963-9029

Sent from my iPhone

> On Mar 2, 2016, at 12:27, Joe Kuan notifications@github.com wrote:
> 
> Oops. I have added the mode=remove. Also fixed the remove mode on static interface
> 
> —
> Reply to this email directly or view it on GitHub.
